Prototope/Animation.swift

```diff
@@ -212,10 +212,10 @@ extension Layer {
 
 // MARK: - Internal interfaces
 
-@objc private class AnimationDelegate: NSObject, POPAnimationDelegate {
+@objc public class AnimationDelegate: NSObject, POPAnimationDelegate {
 	var completionHandler: (() -> Void)?
 
-	func pop_animationDidStop(animation: POPAnimation, finished: Bool) {
+	public func pop_animationDidStop(animation: POPAnimation, finished: Bool) {
```

Comment:
Oooooh, I got bit by so many times! You should be able to keep the class private and make this work by declaring the function as `@objc func pop_animationDidStop`